### PR TITLE
python39Packages.pdfminer: 20220319 -> 20220506

### PR DESCRIPTION
**Pull request by marsam**
###### Description of changes

https://github.com/pdfminer/pdfminer.six/releases/tag/20220506

###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [x] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [22.05 Release Notes (or backporting 21.11 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2205-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Comment by marsam** [comment]
@GrahamcOfBorg build ocrmypdf

**Comment by marsam** [comment]
@GrahamcOfBorg build ocrmypdf

**Comment by marsam** [comment]
@GrahamcOfBorg build ocrmypdf

**Comment by marsam** [comment]
@GrahamcOfBorg build ocrmypdf

**Comment by Artturin** [comment]
> @GrahamcOfBor build ocrmypdf

If you add it to the tests then it'll be automatically built

https://github.com/NixOS/nixpkgs/pull/168921/files

I also recommend adding pythonImportsCheck even if there are tests included

File: pkgs/development/python-modules/ocrmypdf/default.nix [hunk 1]
```diff
@@ -83,7 +83,7 @@ buildPythonPackage rec {
   postPatch = ''
     # https://github.com/ocrmypdf/OCRmyPDF/issues/933
     substituteInPlace setup.cfg \
-      --replace "pdfminer.six!=20200720,>=20191110,<=20211012" "pdfminer.six!=20200720,>=20191110,<=20220319"
+      --replace "pdfminer.six!=20200720,>=20191110,<=20220319" "pdfminer.six!=20200720,>=20191110,<=20220506"
```
Comment (dotlambda) [inline pkgs/development/python-modules/ocrmypdf/default.nix:86]: Why not
```suggestion
      --replace "pdfminer.six!=20200720,>=20191110" "pdfminer.six!=20200720,>=20191110"
```

**Comment by siraben** [comment]
Result of `nixpkgs-review pr 171883` run on aarch64-darwin [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>4 packages marked as broken and skipped:</summary>
  <ul>
    <li>python310Packages.weboob</li>
    <li>python310Packages.woob</li>
    <li>python39Packages.weboob</li>
    <li>python39Packages.woob</li>
  </ul>
</details>
<details>
  <summary>5 packages built:</summary>
  <ul>
    <li>pdfminer (python39Packages.pdfminer)</li>
    <li>python310Packages.pdfminer</li>
    <li>python310Packages.pdfx</li>
    <li>python39Packages.pdfx</li>
    <li>visidata</li>
  </ul>
</details>

**Pull request merged**